src/utils.ts

```diff
@@ -28,3 +31,14 @@ export function make(
 
   return el;
 }
+
+/**
+ * Send user's data to Notion
+ *
+ * @param {string} text - Text for request to database
+ */
+export function sendDataToNotion(text: string): void {
+  const notion = new Notion(exampleConfiguration);
```

Comment:
Do you need to create a notion instance every time? Can you use a singleton for this?